api/v1beta1/moduleimagesconfig_types.go

```diff
@@ -80,8 +80,9 @@ type ModuleImagesConfigStatus struct {
 	ImagesStates []ModuleImageState `json:"imagesStates"`
 }
 
-// +kubebuilder:object:root=true
-// +kubebuilder:subresource:status
+//+kubebuilder:object:root=true
+//+kubebuilder:subresource:status
+//+kubebuilder:resource:scope=Namespaced
```

Comment:
It is already namespaced https://github.com/kubernetes-sigs/kernel-module-management/blob/ef3e3172497a9428014c561cc463ff153cef71da/api/v1beta1/moduleimagesconfig_types.go#L87